api/app/signals/apps/sigmax/views.py

```diff
@@ -65,20 +67,28 @@ def _handle_unknown_soap_action(request):
         status=500)
 
 
+def _parse_sia_id(sia_id):
+    """Extract `id` from `Signal.sia_id` type strings."""
+    if sia_id[:4] == 'SIA-':
```

Comment:
Syntactic sugar, you can also check this with `sia_id.startswith('SIA-')`, which returns a boolean. Same result, a little bit more pythonic.